SplunkRumWorkspace/SplunkRum/SplunkRum/AppLifecycle.swift

```diff
@@ -103,8 +103,10 @@ func lifecycleEvent(_ event: String) {
     // these two attempt to send to the beacon
     if event == UI_APPLICATION_WILL_TERMINATE_NOTIFICATION ||
             event == UI_APPLICATION_DID_ENTER_BACKGROUND_NOTIFICATION {
-        OpenTelemetrySDK.instance.tracerProvider.forceFlush()
 
+        DispatchQueue.global(qos: .userInitiated).async {
```

Comment:
I think background queue would be better here

Reply:
Yes 
func forceFlush(timeout: TimeInterval?)

This method is executed synchronously on the calling thread and Maximum time the flush complete or abort. If nil, it will wait indefinitely.

Reply:
I mean this: https://developer.apple.com/documentation/dispatch/dispatchqos/1780981-background